components/function-controller/cmd/webhook/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"context"
+	"errors"
+
+	"github.com/vrischmann/envconfig"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
+	"knative.dev/pkg/webhook"
+	"knative.dev/pkg/webhook/certificates"
+	"knative.dev/pkg/webhook/resourcesemantics"
+	"knative.dev/pkg/webhook/resourcesemantics/defaulting"
+	"knative.dev/pkg/webhook/resourcesemantics/validation"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	serverlessv1alhpa1 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
+)
+
+var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
+	serverlessv1alhpa1.GroupVersion.WithKind("Function"): &serverlessv1alhpa1.Function{},
+}
+
+type config struct {
+	SystemNamespace    string `envconfig:"default=kyma-system"`
+	WebhookServiceName string `envconfig:"default=serverless-webhook-svc"`
+	WebhookSecretName  string `envconfig:"default=serverless-webhook"`
+	WebhookPort        int    `envconfig:"default=8443"`
+}
+
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;update
+// +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
+// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=mutatingwebhookconfigurations;validatingwebhookconfigurations,verbs=get;list;watch;create;update;patch;delete
+
+func main() {
+	cfg := &config{}
+	if err := envconfig.Init(cfg); err != nil {
```

Comment:
In all our controllers etc. we use the prefix `APP` in the envs that we use in the code, please here use it too.